docs/sdk/auth/fragments/android/working-with-api.md

```diff
@@ -118,7 +118,27 @@ AWSMobileClient.getInstance().signIn(username, password, null, new Callback<Sign
 });
 ```
 
-## Confirm SignIn
+### Confirm SignIn (MFA)
+
+In order to setup multi-factor authentication, choose `Manual configuration` while setting up auth in the CLI and when you get to the `Multifactor authentication` step, choose these values:
```

Comment:
Nit: Run-on sentence. Consider breaking it at the conjunction and reworking for clarity.